x/ibc/04-channel/types/msgs.go

```diff
@@ -389,3 +390,177 @@ func (msg MsgChannelCloseConfirm) GetSignBytes() []byte {
 func (msg MsgChannelCloseConfirm) GetSigners() []sdk.AccAddress {
 	return []sdk.AccAddress{msg.Signer}
 }
+
+// MsgPacket receives incoming IBC packet
+type MsgPacket struct {
+	Packet `json:"packet" yaml:"packet"`
+
```

Comment:
no need for extra spaces on types and constructor

Reply:
fixed